pom.xml

```diff
@@ -3288,6 +3288,7 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.SimplifyBooleanExpressionCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.SimplifyBooleanReturnCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.coding.UnusedCatchParameterShouldBeUnnamedCheck*</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.coding.UnusedLambdaParameterShouldBeUnnamedCheckParameterShouldBeUnnamedCheck*</param>
```

Comment:
https://checkstyle.semaphoreci.com/jobs/b9e32e92-424f-4fc7-ac52-33020a92aaa7

You may want to recheck this.

Reply:
done

Reply:
@mahfouz72 we have a bunch of surviving mutants now

Reply:
They should be resolved now